merlin-driver/src/main/java/gov/nasa/jpl/aerie/merlin/driver/ActivityInstanceId.java

```diff
@@ -0,0 +1,3 @@
+package gov.nasa.jpl.aerie.merlin.driver;
+
+public final record ActivityInstanceId(long id) {}
```

Comment:
Records are `final` by default 👀

Reply:
Fixed. I wasn't sure, but when it didn't complain about me writing final I wrongly assumed that meant final was not the default.